### ISSUE TITLE
devops wiki delete not working

### ISSUE THREAD

**Issue by wi5nia** [Azure/azure-cli-extensions#876]
## Describe the bug

**Command Name**
`az devops wiki delete
Extension Name: azure-devops. Version: 0.10.0.`

**Errors:**
```
list index out of range
Traceback (most recent call last):
pip-install-ygnynjtx\knack\knack\cli.py, ln 206, in invoke
cli\core\commands\__init__.py, ln 560, in execute
cli\core\commands\__init__.py, ln 618, in _run_jobs_serially
cli\core\commands\__init__.py, ln 609, in _run_job
azext_devops\dev\common\exception_handler.py, ln 26, in azure_devops_exception_handler
    reraise(*sys.exc_info())
Temp\pip-install-ygnynjtx\six\six.py, ln 693, in reraise
...
    return 'azdevops-cli:' + normalize_url_for_key(organization)
azext_devops\dev\common\_credentials.py, ln 85, in normalize_url_for_key
    organization_name = organization_name.split('/')[1]
IndexError: list index out of range
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az devops wiki delete --wiki {} --organization {} --project {}`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Shell: cmd.exe

azure-cli 2.0.66 *

Extensions:
azure-devops 0.10.0

```

**Comment by atbagga** [comment]
@wi5nia This looks same as issue https://github.com/Azure/azure-cli-extensions/issues/751
It was fixed in version 0.11.0 (error handling was added for users to be able to figure out the issue.)

**Issue:** You are probably giving organization name instead of url. 

Hopefully that will resolve the issue for you. In case it does not please let me know on this thread.

**Comment by atbagga** [comment]
@wi5nia Can you please confirm the issue is a duplicate and if your issue is resolved.
